src/fides/api/ops/util/cache.py

```diff
@@ -20,6 +23,52 @@
 _connection = None
 
 
+class CustomJSONEncoder(json.JSONEncoder):
+    def default(self, o: Any) -> Any:  # pylint: disable=too-many-return-statements
+        if isinstance(o, Enum):
+            if isinstance(o.value, str):
+                return o.value
+            self.default(o.value)
+        if isinstance(o, bytes):
+            return f"quote_encoded{quote(o)}"
+        if isinstance(o, (datetime, date)):
+            return o.isoformat()
+        if isinstance(o, dict):
+            for _, v in o.items():
+                if not isinstance(v, str) and not isinstance(v, bytes):
+                    self.default(v)
+            return o
+        if isinstance(o, ObjectId):
+            return f"encoded_object_id{str(o)}"
```

Comment:
This looks like it's to handle the Mongo edge case described below?

Reply:
Correct